zilliqa/src/scilla.rs

```diff
@@ -715,7 +718,19 @@ impl ActiveCall {
                 Ok(Some((val, "ByStr20".to_owned())))
             }
             "_codehash" => {
-                todo!()
+                let code_bytes = match &account.account.code {
+                    Code::Evm(bytes) => bytes.clone(),
```

Comment:
Double check the behaviour in ZQ1, but I'd guess we should treat EVM accounts as if they had no Scilla code.

Reply:
It appears that ZQ1 has the same behaviour